skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md

```diff
@@ -937,6 +937,28 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ChannelRecordingDownload
+
+Controls how channel meeting recordings are saved, permissioned, and who can download them. 
+
+Possible values:
+
+- Allow - Saves channel meeting recordings to a "Recordings" folder in the channel. The permissions on the recording files will be based off the Channel SharePoint permissions. This is the sames as any other file uploaded for the channel.
+- Block - Saves channel meeting recordings to a "Recordings\View only" folder in the channel. Channel owners will have full rights on the recordings in this folder, but channel members will have read access without ability to download. 
```

Comment:
```suggestion
Controls how channel meeting recordings are saved, permissioned, and who can download them. 

Possible values:

- Allow - Saves channel meeting recordings to a "Recordings" folder in the channel. The permissions on the recording files will be based on the Channel SharePoint permissions. This is the same as any other file uploaded for the channel.
- Block - Saves channel meeting recordings to a "Recordings\View only" folder in the channel. Channel owners will have full rights to the recordings in this folder, but channel members will have read access without the ability to download. 
```

Reply:
Yes it will be included in the next GA version of the module, it's only in the Preview version today. We are going to be telling our customers to use the preview version for now, as this change is an important feature they've been waiting for as part of the change over from Teams Meeting Recordings being saved to ODB/SPO instead of Stream (classic).